.Rbuildignore

```diff
@@ -16,3 +16,6 @@
 ^revdep$
 ^\.github$
 ^LICENSE\.md$
+^forcats\.Rcheck$
```

Comment:
It's not a big deal to add these, but it suggests something might be wonky with your process, as they should end up in the parent directory, not here.

Reply:
Ok! I'm running it from RStudio Cloud. Thank you for the tip!